rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -201,15 +202,7 @@ namespace rocal{
         rocalGetImageId(context,ptr);
         return py::cast<py::none>(Py_None);
     }
-    py::object wrapper_labels_BB_count_copy(RocalContext context, py::array_t<int> array)
 
```

Comment:
Are the other wrappers required? If not we can remove them?

Reply:
Commented the wrappers out